### PR TITLE
Add ep.33

### PR DESCRIPTION
**Pull request by PercyDan54**
- [x] Pending timing pass

### CODE REVIEW

File: S3/English/ASS_33.ass [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Dialogue: 0,0:03:47.72,0:03:50.35,Default,,0,0,0,,I'm already happy if you make less trouble
+Dialogue: 0,0:03:50.64,0:03:51.30,Default,,0,0,0,,Lynn
+Dialogue: 0,0:03:51.45,0:03:52.83,Default,,0,0,0,,You are so busy this year
+Dialogue: 0,0:03:52.93,0:03:54.05,Default,,0,0,0,,We won't do this in the future
+Dialogue: 0,0:03:54.05,0:03:55.00,Default,,0,0,0,,Don't say that if you can't do it
+Dialogue: 0,0:03:56.76,0:03:58.10,Default,,0,0,0,,Could we...
+Dialogue: 0,0:04:01.71,0:04:03.35,Default,,0,0,0,,Let's make an exception today
+Dialogue: 0,0:04:03.40,0:04:04.75,Default,,0,0,0,,Hooray
+Dialogue: 0,0:04:16.79,0:04:18.18,Default,,0,0,0,,It's their happiness
+Dialogue: 0,0:04:18.25,0:04:19.30,Default,,0,0,0,,I got nothing
```
Comment (PercyDan54) [inline S3/English/ASS_33.ass:107]: Official subs has mistimed line 107, should I follow the vocal or their sub? Currently I did the latter

File: S3/English/ASS_33.ass [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Dialogue: 0,0:03:13.34,0:03:14.34,Default,,0,0,0,,Happy birthday Lynn
+Dialogue: 0,0:03:14.45,0:03:15.55,Default,,0,0,0,,Here's a diary of this year
+Dialogue: 0,0:03:15.79,0:03:17.26,Default,,0,0,0,,Two-thirds of the year has passed
+Dialogue: 0,0:03:17.38,0:03:17.77,Default,,0,0,0,,Yeah
+Dialogue: 0,0:03:18.23,0:03:19.77,Default,,0,0,0,,So I ripped them off for you
+Dialogue: 0,0:03:20.00,0:03:21.15,Default,,0,0,0,,Happy birthday Lynn
+Dialogue: 0,0:03:21.45,0:03:22.53,Default,,0,0,0,,A dog food bowl for you
+Dialogue: 0,0:03:22.64,0:03:23.78,Default,,0,0,0,,Isn't this your bowl
+Dialogue: 0,0:03:24.56,0:03:25.78,Default,,0,0,0,,Lynn, this hell dirt...
+Dialogue: 0,0:03:25.89,0:03:26.63,Default,,0,0,0,,Leave it for yourself
```
Comment (Hrshl7) [inline S3/English/ASS_33.ass:86]: "Keep it" would make more sense

File: S3/English/ASS_33.ass [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+Dialogue: 0,0:03:27.35,0:03:27.85,Default,,0,0,0,,Out of the way
+Dialogue: 0,0:03:28.25,0:03:28.85,Default,,0,0,0,,Lynn
+Dialogue: 0,0:03:29.28,0:03:30.52,Default,,0,0,0,,Here's the new clothes for you
```
Comment (Hrshl7) [inline S3/English/ASS_33.ass:89]: I say drop the "the"
Abu came up with bandages, but adding a "the" would mean that we already know about which clothes or that it doesn't come from the person giving them to him

File: S3/English/ASS_33.ass (hunk 1 above)
Comment (Hrshl7) [reply S3/English/ASS_33.ass:107]: Depends on how long it is (bc of character/seconds) and if it's not too mistimed (as in moved enough for the viewer not to understand which written line correspond to the one being said)
Personally, I always fit my subs to the official ones but bc it bothers me to have them not appear and disappear exactly at the same time.

**Review by Hrshl7 — COMMENTED** [review]
(Lynn is a mood)

File: S3/English/ASS_33.ass (hunk 1 above)
Comment (PercyDan54) [reply S3/English/ASS_33.ass:107]: > Depends on how long it is (bc of character/seconds) and if it's not too mistimed (as in moved enough for the viewer not to understand which written line correspond to the one being said) Personally, I always fit my subs to the official ones but bc it bothers me to have them not appear and disappear exactly at the same time.

It's mistimed for about 0.6s

File: S3/English/ASS_33.ass [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+Dialogue: 0,0:03:11.04,0:03:12.34,Default,,0,0,0,,Give anything we have
+Dialogue: 0,0:03:13.34,0:03:14.34,Default,,0,0,0,,Happy birthday Lynn
+Dialogue: 0,0:03:14.45,0:03:15.55,Default,,0,0,0,,Here's a diary of this year
+Dialogue: 0,0:03:15.79,0:03:17.26,Default,,0,0,0,,Two-thirds of the year has passed
+Dialogue: 0,0:03:17.38,0:03:17.77,Default,,0,0,0,,Yeah
+Dialogue: 0,0:03:18.23,0:03:19.77,Default,,0,0,0,,So I ripped them off for you
+Dialogue: 0,0:03:20.00,0:03:21.15,Default,,0,0,0,,Happy birthday Lynn
+Dialogue: 0,0:03:21.45,0:03:22.53,Default,,0,0,0,,A dog food bowl for you
+Dialogue: 0,0:03:22.64,0:03:23.78,Default,,0,0,0,,Isn't this your bowl
+Dialogue: 0,0:03:24.56,0:03:25.78,Default,,0,0,0,,Lynn, this hell dirt...
```
[the remark this answers is not in the document]
Comment (PercyDan54) [reply S3/English/ASS_33.ass:85]: image
This is what I found

[the remark this answers is not in the document]
Comment (Hrshl7) [reply S3/English/ASS_33.ass:85]: by looking at the episode it does have a face so maybe go for "hell cleanser gel" or just "hell cleanser" in order to be consistent?

File: S3/English/ASS_33.ass (hunk 1 above)
Comment (Hrshl7) [reply S3/English/ASS_33.ass]: So I looked at the episode and it's not mistimed enough for it to be that ununderstandable, so if by following the vocal it ends up being on the next line, follow the sub

**Pull request merged**